CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+### 2.31.1
+This release includes:
+* Fluent Bit [1.9.10](https://fluentbit.io/announcements/v1.9.10/)
+* Amazon CloudWatch Logs for Fluent Bit 1.9.1
+* Amazon Kinesis Streams for Fluent Bit 1.10.1
+* Amazon Kinesis Firehose for Fluent Bit 1.7.1
+
+Compared to `2.31.0`, this release adds the following feature that we are working on getting accepted upstream:
+* Feature - Add a configuration option for S3 retry attempts [fluent-bit:6475](https://github.com/fluent/fluent-bit/pull/6475)
+* Bug - Resolve Datadog buffer resize segfault by including critical parts of reverted Datadog PR [aws-for-fluent-bit:491](https://github.com/aws/aws-for-fluent-bit/issues/491)
```

Comment:
I think this is not the most user friendly message. Can we rephrase it to not be how we accomplished this, but what it means for customers?

Reply:
Updated